src/libraries/System.IO.Compression.Brotli/ref/System.IO.Compression.Brotli.cs

```diff
@@ -6,9 +6,10 @@
 
 namespace System.IO.Compression
 {
-    public sealed class BrotliCompressionOptions
+    public sealed partial class BrotliCompressionOptions
     {
-        public int Quality { get; set; }
+        public BrotliCompressionOptions() { }
```

Comment:
This is fine - the metadata would still have this, but good to keep the source in sync with what's emitted from GenAPI.